.github/actions/setup-env/action.yaml

```diff
@@ -70,7 +70,8 @@ runs:
       shell: bash
       if: ${{ inputs.setup-poetry == 'true' }}
       run: |
-        python -m pip install poetry poetry-dynamic-versioning
+        python -m pip install poetry
+        poetry self add "poetry-dynamic-versioning[plugin]"
```

Comment:
This comment applied to the general approach of installing `poetry-dynamic-versioning`. What is the advantage of using `poetry self` vs `pip install` for installing `"poetry-dynamic-versioning[plugin]"`? I tried digging through `poetry-dynamic-versioning` docs and there's not much mention of this. Generally I wonder if it might be simpler to install `poetry-dynamic-versioning` with pip unless there's a strong reason not to. When I test installing both poetry and poetry-dynamic-version it seems like build is able to ascertain the correct dynamic version for Pycytominer.

Reply:
When I tested `pip install poetry poetry-dynaming-versioning[plugin]` , I got an error message when I tried to run `poetry build`. I don't recall the specific message but I think it was something along the lines of "missing command handles". After that I switched to following the instructions in the pdv documentation which state `poetry self add` is the recommended approach. 🤷

Reply:
Thanks for addressing this! It does seem like PDV prefers the `poetry self` command over `pip install`. In practice I haven't been able to tell the difference (each seem to act the same, at least now they do). From an understandability I found myself wondering whether `pip install` or `poetry self` commands were more likely to change or be understood long-term. It could go either way really, as PDV too could opt to change how installation works. Seems like following the docs from PDV is a good bet that someone can find the "most correct" way of installing it.